### DISCUSSION TITLE
Date picker

### DISCUSSION THREAD

**Discussion by omniith** [livewire/flux#519] [Feature Requests]
Hi team,

This might be too much to ask, but i'm shooting the shot!
FLUX DATE PICKER pretty please #googlyEyes

Right now its either i'm stuck with basic 2 date inputs for start_date and end_date. 
If i move to use other library like flatpickr, the css is just totally different.

**Comment by ju5t** [comment]
You're not alone, see: #144. Probably best to keep everything in the same discussion.
